packages/components/src/layouts/Navigation/Tab/TabBar/DesktopTabItem.tsx

```
@@ -116,28 +145,32 @@ export function DesktopTabItem(
-                  size="small"
-                  icon="DotHorOutline"
-                  variant="tertiary"
-                  focusVisibleStyle={undefined}
-                  p="$0.5"
-                  m={-3}
-                  testID="browser-bar-options"
-                />
-              ) : null
-            }
             sections={actionList}
+            renderTrigger={<></>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)